Source/WebKit/UIProcess/gtk/WebPageProxyGtk.cpp

```diff
@@ -49,7 +49,8 @@ void WebPageProxy::platformInitialize()
 
 GtkWidget* WebPageProxy::viewWidget()
 {
-    return static_cast<PageClientImpl&>(pageClient()).viewWidget();
+    RefPtr pageClient = this->pageClient();
+    return pageClient ? static_cast<PageClientImpl&>(*pageClient).viewWidget() : nullptr;
```

Comment:
Can we use downcast while we're at it?

Reply:
It's a bit of a weird one, a PageClient is always a PageClientImpl. However, each port has its own `PageClientImpl` subclass (with the same name). `downcast<>()` is great when a class has several subclasses that can co-exist. However, it's not the case here.

I don't mind making this change since getting rid of `static_cast<>()` in general is positive but I'm not sure how to best do the change (what will the type check look like for example). I'm not making this change in this PR for now but I can follow-up if we come up with something we like.